src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/utils.jsx

```diff
@@ -2,17 +2,20 @@ import {pick, pickBy, identity} from 'lodash';
 
 import {defined} from 'app/utils';
 import {getUtcToLocalDateObject} from 'app/utils/dates';
+import {getParams} from 'app/views/events/utils/getParams';
```

Comment:
Should this function be moved to live with the other global selection functions?

Reply:
I'll see if I can move `getParams` to this file.